manifests/run_service.pp

```diff
@@ -4,7 +4,6 @@
 # @api private
 class nomad::run_service {
   $init_selector = $nomad::init_style ? {
-    'launchd' => 'io.nomad.daemon',
     default   => 'nomad',
   }
```

Comment:
This can really be now.

Reply:
@ekohl seems some of your answer is missing

Reply:
Derp. I meant to remove `really` and not `simplified`.